e2e/configmap.go

```diff
@@ -68,14 +68,14 @@ func createConfigMap(pluginPath string, c kubernetes.Interface, f *framework.Fra
 	if err == nil {
 		_, updateErr := c.CoreV1().ConfigMaps(cephCSINamespace).Update(context.TODO(), &cm, metav1.UpdateOptions{})
 		if updateErr != nil {
-			return updateErr
+			return fmt.Errorf("failed to update %w", updateErr)
```

Comment:
failed to update configmap?

Reply:
Yes.  Earlier there was no message so I just added. Will update elsewhere as well.